### PR TITLE
Add support for preventDrift field in ConfigManagement Fleet-level default config

### PR DESCRIPTION
**Pull request by rramkumar1**
**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
gkehub: added `prevent_drift` field to ConfigManagement `fleet_default_member_config`
```

### CODE REVIEW

**Review by melinath — CHANGES_REQUESTED** [review]
Please add this field to tests: https://googlecloudplatform.github.io/magic-modules/develop/test/test/

**Comment by rramkumar1** [comment]
> Please add this field to tests: https://googlecloudplatform.github.io/magic-modules/develop/test/test/

Done

File: mmv1/third_party/terraform/services/gkehub2/resource_gke_hub_feature_test.go.erb [hunk 1]
```diff
@@ -488,6 +488,7 @@ resource "google_gke_hub_feature" "feature" {
   fleet_default_member_config {
     configmanagement {
       version = "1.16.1"
+      prevent_drift = true
       config_sync {
```
Comment (melinath) [inline mmv1/third_party/terraform/services/gkehub2/resource_gke_hub_feature_test.go.erb:492]: This should be inside config_sync (which is why you're getting an alert about missing tests).

 ```suggestion
      config_sync {
        prevent_drift = true
```

**Review by melinath — CHANGES_REQUESTED** [review]
Note to self: I'll need to manually run the test for this after it's fixed since it skips VCR.

**Comment by melinath** [comment]
manual test run: https://hashicorp.teamcity.com/buildConfiguration/TerraformProviders_GoogleCloud_GOOGLE_BETA_MMUPSTREAMTESTS_GOOGLEBETA_PACKAGE_GKEHUB2/160266?hideProblemsFromDependencies=false&hideTestsFromDependencies=false

**Review by melinath — APPROVED** [review]
Manual tests passed - LGTM

**Pull request merged**